libc/libc.go

```diff
@@ -1,16 +1,29 @@
 // Copyright The OpenTelemetry Authors
 // SPDX-License-Identifier: Apache-2.0
 
-package tpbase // import "go.opentelemetry.io/ebpf-profiler/tpbase"
+package libc // import "go.opentelemetry.io/ebpf-profiler/libc"
 
 import (
 	"debug/elf"
 	"fmt"
 	"regexp"
 
+	"go.opentelemetry.io/ebpf-profiler/internal/log"
+
 	"go.opentelemetry.io/ebpf-profiler/libpf/pfelf"
 )
 
+type LibcInfo struct {
+	DTVInfo *DTVInfo
+	TSDInfo *TSDInfo
```

Comment:
nit: remove the pointer indirection here. having this as pointer may actually crash existing code if `TSDInfo` is `nil` and its dereferenced still unconditionally. Keeping these inline also reduce object scatter.